.unibeautifyrc.yml

```
@@ -0,0 +1,24 @@
+---
+JSON:
+  beautifiers: ["Prettier"]
+  indent_char: " "
+  indent_size: 2
+JavaScript:
+  beautifiers: ["Prettier"]
+  indent_char: " "
+  indent_size: 2
+TypeScript:
+  beautifiers: ["Prettier"]
+  align_assignments: false
+  arrow_parens: "as-needed"
+  break_chained_methods: true
+  end_with_comma: true
+  end_with_semicolon: true
+  indent_char: " "
```

Should be using `indent_style` not `indent_char`: https://unibeautify.com/docs/option-indent-style.html

```suggestion
  indent_style: "space"
```